skl/ag7804/src/main/java/se/inera/intyg/common/ag7804/v1/model/converter/InternalToCertificate.java

```diff
@@ -1067,6 +1072,7 @@ private static boolean hasRenewalRelation(Relation relation) {
             return Collections.emptyList();
         }
         return sickLeaves.stream()
+            .filter(item -> item.getPeriod().isValid())
```

Comment:
Coult item.getPerid() be null?

Reply:
Good question! I will look into it before merging!

Reply:
It was easy to do with a unit test so it could probably happen.

Reply:
Fixed